packages/nimble-components/src/wafer-map/specs/features/rendering.md

```diff
@@ -91,37 +93,51 @@ class WaferData {
+-   A row based format that aligns well with the existing public api
+-   Well supported and tested format
+-   Nice public API to use, we don't have to invent a new format, just document our schema for the arrow tables
+-   Designed for large dataset visualizations
+
+In order to choose from these alternatives we will prototype and check:
+
+-   Does it have comparable memory performance
+-   Does it perform well or have significant overhead
+-   Is it easy to divide and use in parallel
 
 ### Rendering
 
-The rendering will also be changed to take place inside a worker thread. The worker will be responsible for parsing each layer of the data set and it will render the information contained with the specific color codes and opacity.
+An alternate renderer inside a worker thread will be created to live in parallel in the wafer-map :
 
-We will be using the [comlink](https://github.com/GoogleChromeLabs/comlink) library to spin up the worker and communicate with it.
+-   During development the data assigned to the dies property will determine the renderer to use. The current WaferMapDie type will choose the current renderer and the new WaferDataFormat will pick the new renderer.
+-   When development is complete the old renderer will be removed and the previous WaferMapDie format will be supported and map to the new format internally.
 
-The worker code will be precompiled by rollup in a library and sent as a Blob to the worker.
+The render worker:
 
-The worker will function as a state automaton which will need some data after initialization, and it will render everything using an [OffscreenCanvas](https://developer.mozilla.org/en-US/docs/Web/API/OffscreenCanvas).
+-   will be responsible for parsing each layer of the data set and it will render the information contained with the specific color codes and opacity.
+-   will be using the [comlink](https://github.com/GoogleChromeLabs/comlink) library to communicate with the main thread.
+-   will be precompiled by rollup in a library and sent as a Blob to the worker initializer. Workers normally require a separate script file that is downloaded dynamically to run. To avoid the need for dynamically fetched resources and enable static compilation the worker code will be precompiled by rollup in a library and sent as a Blob to the worker.
+-   will function as a state automaton which will need some data after initialization, and it will render everything using an [OffscreenCanvas](https://developer.mozilla.org/en-US/docs/Web/API/OffscreenCanvas). `SharedArrayBuffers` can be used to efficiently share memory buffers across multiple web workers but require strict server configuration to be enabled. As such they will not be leveraged and are not needed to meet the performance targets.
```

Comment:
Please include the link that was provided describing the needed server config

```suggestion
-   will function as a state automaton which will need some data after initialization, and it will render everything using an [OffscreenCanvas](https://developer.mozilla.org/en-US/docs/Web/API/OffscreenCanvas). `SharedArrayBuffers` can be used to efficiently share memory buffers across multiple web workers but require [strict server configuration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer#security_requirements) to be enabled. As such they will not be leveraged and are not needed to meet the performance targets.
```